packages/time-picker/src/TimePicker/components/Button/Button.spec.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import { Button } from "./Button"
+
+const ButtonWrapper = (customProps?): JSX.Element => (
+  <Button
+    exampleRequiredString="Hello!" /** @todo: Add default values for your required props (override them with customProps if needed) */
+    {...customProps}
+  />
+)
+
+describe.skip("<Button />", () => {
```

Comment:
Pointless test, should this just be deleted?

Reply:
Ah sorry these tests are WIP, I never got around to writing them. 

Not sure these particularly need unit tests, so may end up just deleting them